core/base/src/TSystem.cxx

```diff
@@ -4310,9 +4310,9 @@ TString TSystem::SplitAclicMode(const char *filename, TString &aclicMode,
          arguments = "";
          delete []fname;
          return "";
-      } else if (s2) {
+      } else if (s2 && s2 > fname) {
```

Comment:
This is an error/unlikely case anyway, but the following might be more semantically consistent:
```
      } else if (s2) {
         if (s2 > fname) {
            // Skip/trim spaces
            s2--;
            while (s2 > fname && *s2 == ' ') s2--;
            s2++;
      }
```